start-dev.sh

```
@@ -13,6 +13,19 @@ YELLOW='\033[1;33m'
 RED='\033[0;31m'
 NC='\033[0m' # No Color
 
+# .env 파일 로드
+if [ -f ".env" ]; then
+    echo -e "${BLUE}📄 .env 파일을 로드합니다...${NC}"
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)